### ISSUE TITLE
Implement ad tracking snippet 

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1056]
# TL;DR

We minimize the amount of tracking we do on Lookit, but it appears to be no longer possible to set up a google ad campaign without some form of 'conversion metric' that depends on behavior on the site you direct people to.  So, to make the new ad campaign work, we need to insert the snippet below into those URLs that will be tracked by the google ad campaign. 

# Acceptance Criteria

- [ ] Prior to implementation, write up a short (1-2 paragraph MAX) description of what the code does, and how it interacts with a participant's computer. Edit this issue so we have a record of this in case we need to roll back, or in case anyone asks about this.
- [ ] Add the code snippet in the <head> of the following URLS (which are the plausible landing page or endpoint for conversion tracking):

- https://lookit.mit.edu/
- https://lookit.mit.edu/signup/
- https://lookit.mit.edu/account/demographics/
- https://lookit.mit.edu/account/children/
- https://lookit.mit.edu/studies/

# Implementation Notes
Here's the code snippet: 

```
<!-- Google tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-XMVJEVKPVV">
</script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-XMVJEVKPVV');
</script>
```
